cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -87,6 +100,9 @@ spec:
       - name: kube-proxy
         configMap:
           name: kube-proxy
+      - name: xtables-lock
+        hostPath:
+          path: /run/xtables.lock
```

Comment:
Is this a file or directory? I suppose file. Will k8s always mount it as a file as well?